### PR TITLE
Hg 668 wiki redirect

### PR DESCRIPTION
**Pull request by kenkouot**
Fixes /wiki redirecting to /wiki/wiki

### CODE REVIEW

File: server/facets/operations/redirectToRoot.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/// <reference path='../../../typings/hapi/hapi.d.ts' />
+
+function redirectToRoot (request: Hapi.Request, reply: any) {
```
Comment (lizlux) [inline server/facets/operations/redirectToRoot.ts]: `:void` ;)

File: server/routes.ts [hunk 2]
```diff
@@ -67,6 +67,11 @@ unauthenticatedRoutes = [
 		path: '/heartbeat',
 		handler: require('./facets/operations/heartbeat')
 	},
+	{
+		method: 'GET',
+		path: '/wiki',
```
Comment (rogatty) [inline server/routes.ts]: Ticket is about `/wiki/`. Is this enough to fix the problem?

Comment (lizlux) [inline server/routes.ts]: nope, was just testing that. `/wiki` redirects to home page. `/wiki/` redirects to `/wiki/wiki/` still.

Comment (kenkouot) [inline server/routes.ts]: fixed

File: server/app.ts [hunk 3]
```diff
@@ -22,7 +22,13 @@ var counter = 1,
 	isDevbox: boolean = localSettings.environment === Utils.Environment.Dev,
 	plugins: any,
 	/* Creates new `hapi` server */
-	server = new Hapi.Server();
+	server = new Hapi.Server({
+		connections: {
+			router: {
+				stripTrailingSlash: true
```
Comment (rogatty) [inline server/app.ts]: Does `/` still redirect to the main page? I wonder if there are some use cases that this option can break... Nothing comes to my mind right now.

Comment (kenkouot) [inline server/app.ts]: Still works

**Comment by rogatty** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
Found defect: https://wikia-inc.atlassian.net/browse/HG-693

**Pull request merged**